### DISCUSSION TITLE
How to test typescript (api) functions?

### DISCUSSION THREAD

**Discussion by mariosant** [vercel/community#281] [Help] [answered]
Hi there, I currently have a project that consists of a few serverless typescript functions and I would like to test these.
Is there any suggested way to do that?

**Comment by samcx** [comment, marked as answer]
Hi there!

If you are using Next.js, you can simply use the command `next dev`. If you are not using Next.js, you can use the `vercel dev` command with our Vercel CLI → https://vercel.com/docs/cli#commands/dev.
